python/tvm/relay/frontend/tensorflow.py

```diff
@@ -1378,9 +1378,11 @@ def _gather_nd():
     def _impl(inputs, attr, params, mod):
         indices_dims = len(_infer_shape(inputs[1], mod))
         indices = _op.transpose(inputs[1], axes=[-1] + list(range(indices_dims-1)))
+        attr_new = {}
+        attr_new['one_dim_support'] = True
```

Comment:
Looks like ```one_dim_support``` is just for checking but doesn't change functionality. Can you explain a bit more about why we need this?

Reply:
@kevinthesun : Sorry for late reply! Actually the difficulty here is Gather_ND has different behavior in MXNet & Tensorflow. 
In MXNet it supports only Dim >= 2, but in Tensorflow it supports Dim >= 1.
So if we just check Input dims, it may not help the Op works in both the case.
That is the reason i added additional attribute to control it.
But we can do it another way as well, like adding check in MXNet frontend minimum 2-dim, and modify check in Op code to support minimum 1-dim.

In that way also we can solve for both. We can discuss on it, which one will be better. Please let me know your valuable opinion, Thanks!

Reply:
Thanks for clarification. IMO, if topi gather_nd has capability to handle  >= 1d cases, it would be better to put the decision on the frontend.